CONTRIBUTING.md

```diff
@@ -118,30 +126,8 @@ Please take a moment to review this document in order to make the contribution p
     - See [Creating a pull request from a fork](https://help.github.com/articles/creating-a-pull-request-from-a-fork/) for step-by-step instructions.
     - Ensure the base branch is assigned to `master`.
 
-### NPM commands
-
-We use specific NPM scripts to test, lint, and build our demo and distributed files. Here are the scripts to choose from:
-
-- `npm run demo:server` to run the demo server on port 9090 and watch for changes
-- `npm run demo:build` to build the demo directory in `carbon-charts/demo/`
-- `npm run build` to build the distributed directory in `carbon-charts/dist/`
-- `npm run lint` to run TS lint on the source files
-- `npm run test` to run unit tests on the source files
-
 ## Code guidelines
-
```

Comment:
Why not keep some relevant information and resources?

Feel free to reword as you see fit.